web/html/src/branding/css/base/theme.scss

```diff
@@ -1312,3 +1312,17 @@ tr.changed > td {
     font-size: .4em;
   }
 }
+ :root {
```

Comment:
We want to get rid of the large theme.scss file over time, not add to it. Please lift this out into a separate module as other code does.